orga/public/robots.txt

```diff
@@ -1,4 +1,3 @@
 # http://www.robotstxt.org
 User-agent: *
-Disallow: /
-Noindex: /
+Allow: /
```

Comment:
**TO DO :** allow uniquement sur la page de connexion ⚠️  ou preciser quel robot laisser passer